packages/space-header/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { inject_to_body} from "./inject_to_body"
```

Comment:
maybe we could add a prettifier to have unified spaces / semicolons, but that's just code-smetics

Reply:
The code was on another project before (without prettier), so should be good now!